index.html

```diff
@@ -2,7 +2,10 @@
 layout: default
 title: PowerPointLabs
 ---
-
+<style>
+	.testDiv{ width:730px; height:80px} 
```

Comment:
1. don't write css here. all css codes are put into main.less, so that the style can be managed in one place.
2. in order to make it maintainable, pls give it a meaningful name (like testimonial-row or testimonial-container); if this element is unique (cannot appear more than once), use id instead of class.